string objectName
---
int64 success_bool
